file-formats/ntta/type/zif_aff_ntta_v1.intf.abap

```diff
@@ -0,0 +1,52 @@
+INTERFACE zif_aff_ntta_v1
+  PUBLIC.
+
+  TYPES:
+    ty_notetype TYPE c LENGTH 20,
+
+    "! <p class="shorttext">General Information</p>
+    "! General information
+    BEGIN OF ty_general_information,
+      "! <p class="shorttext">SAP Object Node Type</p>
+      "! SAP Object Node Type
+      "! $required
+      object_node_type    TYPE zif_aff_types_v1=>ty_object_name_30,
+      "! <p class="shorttext">Implementation Exit Class</p>
+      "! Class implementing the exits for authorization and feature control
+      "! $required
+      implementation_exit TYPE zif_aff_types_v1=>ty_object_name_30,
+    END OF ty_general_information,
+
+    "! <p class="shorttext">Notetype</p>
+    "! Notetype
+    BEGIN OF ty_notetype_structure,
+      "! <p class="shorttext">Notetype</p>
+      "! Notetype
+      "! $required
+      notetype TYPE ty_notetype,
```

Comment:
@schneidermic0 
Currently we cannot rule out that we will add addition fields. So I don't think it would hurt to leave it that way. Are there reasons why we should simplify it?

Reply:
The only reason would be, that it is simpler/more lightweight in the JSON. ;) 
Otherwise, it's fine. I will mark the conversation as resolved.